packages/editor/styles/styles.css

```diff
@@ -54,10 +50,11 @@
 }
 
 .ProseMirror p code {
-  background-color: var(--background);
+  background-color: var(--background-secondary);
   border: 1px solid var(--border);
```

Comment:
Can you try without the border? I think it'll look better.

Reply:
without border

<img width="863" height="300" alt="image" src="https://github.com/user-attachments/assets/bc6ef8d1-9b56-4ff8-b189-6a881f045e82" />

Reply:
Or we could change the font size but that'd look odd.